cypress/e2e/signUp/signUp.cy.ts

```diff
@@ -0,0 +1,53 @@
+const signUpPageTestIds = {
+  inputName: 'sign-up-form-name',
+  inputEmail: 'sign-up-form-email',
+  inputPassword: 'sign-up-form-password',
+  submitButton: 'sign-up-form-submit',
+};
+
+const signUpAriaLabels = {
+  loading: 'sign-up-form-loader',
+};
+
+describe('Sign Up', () => {
+  beforeEach(() => {
+    cy.task('resetAll');
+  });
+
+  describe('given valid credentials', () => {
+    it('creates user successfully', () => {
+      cy.visit('/auth/signup');
+      cy.findByTestId(signUpPageTestIds.inputName).type('Dev Nimble');
+      cy.findByTestId(signUpPageTestIds.inputEmail).type('dev@nimblehq.co');
+      cy.findByTestId(signUpPageTestIds.inputPassword).type('password');
+
+      cy.findByTestId(signUpPageTestIds.submitButton)
+        .invoke('click')
+        .hasActiveSpinner(signUpAriaLabels.loading);
+
+      cy.hasToastMessage(
+        'success',
+        'The user account has been successfully created. You can now log in.'
+      );
+    });
+  });
+
+  describe('given email is NOT unique', () => {
+    it('throws error', () => {
+      const email = 'dev@nimblehq.co';
+      const password = 'password';
+      cy.visit('/auth/signup');
```

Comment:
Let's insert a line break to separate the `arrange` from the `act`(as in `Arrange/Act/Assert (AAA)`) ✋ 

```suggestion
      const password = 'password';
      
      cy.visit('/auth/signup');
```